styles/05-objects/card.css

```diff
@@ -0,0 +1,163 @@
+/* .cards, */
+
+.card-container,
+.card-primary-content,
+.actions {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.actions {
+  flex-wrap: nowrap;
+}
+
+.cards {
+  padding: 10px;
+  justify-content: space-around;
+  gap: 4rem;
+}
+
+.card-container {
+  background-color: white;
+  box-shadow: 0 0 5px rgb(58, 57, 58);
+  row-gap: 5px;
+  width: 400px;
+  flex-direction: column;
+  border-radius: 15px 15px 20px 20px;
+}
+
+.card-primary-content {
+  flex-direction: column;
+}
+
+.card-primary-content:hover {
+  filter: grayscale(0.1) invert(0.1);
+  background-color: #fff;
+  backdrop-filter: contrast(100%);
+  transform: 0.3s;
+}
+
+.card-header {
+  margin: 1rem 0 0.5rem
+}
+
+.card-header,
+.card-body {
+  padding: 0 15px;
+}
+
+.decription {
+  margin: 0.5rem 0;
+}
+
+.media img,
+.card-primary-content:hover {
+  max-width: 100%;
+  border-top-left-radius: 15px;
+  border-top-right-radius: 15px;
+}
+
+.actions {
+  align-items: center;
+  justify-content: space-around;
+  position: relative;
+  margin: 2ch;
+}
+
+
+.actions button {
+  margin: 0 10px;
+  min-width: 50px;
+  min-height: 50px;
+  border: none;
+}
+
+
+
+
+
+.like {
```

Comment:
Share/like should be part of social media css, not generic cards.css in my opinion. I could be wrong.